package.json

```diff
@@ -25,6 +25,7 @@
   "author": "Erin Depew",
   "license": "MIT",
   "dependencies": {
+    "immutable": "^3.8.1",
```

Comment:
Oh neat.  I had to dig into what redux meant by immutables, but I see where it will be important with keeping the remote and local state in sync.